src/components/ground-nav/demo/organization.json

```diff
@@ -1,14 +1,14 @@
 {
   "name": "Cloud Four, Inc.",
   "address": {
-    "streetAddress": "510 SW 3rd Ave, Suite 420",
-    "addressLocality": "Portland",
-    "addressRegion": "Oregon",
-    "postalCode": "97204",
-    "addressCountry": "USA"
+    "street_address": "510 SW 3rd Ave, Suite 420",
+    "address_locality": "Portland",
+    "address_region": "Oregon",
+    "postal_code": "97204",
+    "address_country": "USA"
   },
   "email": "info@cloudfour.com",
   "telephone": "+1 (503) 290-1090",
   "url": "https://cloudfour.com/",
-  "foundingDate": "2007-12-13"
+  "founding_date": "2007-12-13"
```

Comment:
Non-blocking question: should we still be using camelCase in JSON? Or is that more confusing than using snake case in both places? (I assume JS will continue to use camelCase)

Reply:
I think in this file it makes sense to use underscores, because this file only exists as data to plug into the twig templates

Reply:
> I assume JS will continue to use camelCase

Yes, this is the expectation.

>  I think in this file it makes sense to use snake_case, because this file only exists as data to plug into the twig templates

👍